### PR TITLE
[CIRCLE-3453] document no_output_timeout

### PR DESCRIPTION
**Pull request by notnoopci**
`run` step can take a `no_output_timeout` modifier

### CODE REVIEW

File: jekyll/_cci2/configuration-reference.md [hunk 1]
```diff
@@ -190,6 +190,7 @@ shell | N | String | Shell to use for execution command (default: `/bin/bash -e`
 environment | N | Map | Additional environmental variables, locally scoped to command
 background | N | Boolean | Whether or not this step should run in the background (default: false)
 working_directory | N | String | What directory to run this step in (default:  [`working_directory`](#jobs) of the job)
+no_output_timeout | N | String | Time duration where the command can run without output. The is decimal number with unit suffix, such as "20m", "1h", "5s" (default: 10 minutes)
```
Comment (felicianotech) [inline jekyll/_cci2/configuration-reference.md]: "The is decimal number" <- small grammar error there

**Comment by felicianotech** [comment]
@notnoopci I pointed out a small grammar error. I could fix it myself but, more importantly, you mention decimal but all the examples are exclusively integers. Are decimal values such as `2.5m` allowed? If so, that should be an example. Otherwise I think we should say integers to be more specific.

**Comment by notnoopci** [comment]
That should have been `The string is ...` - and yes decimal is ok so `2.5m` would work.  Updated example.

**Comment by felicianotech** [comment]
LGTM. Merging after passed test.

**Pull request merged**